htdocs/js/modules/direct_entry.js

```diff
@@ -63,7 +63,13 @@ $(document).ready(function() {
             value: nextPage
         }).appendTo("#test_form");
 
-        formEl.action = document.documentURI;
+        if (document.documentURI) {
+            var documentURI=document.documentURI;
```

Comment:
Please move the variable declaration at the top of the function (to prevent any issue whit Hoisting) and use `let` instead of `var`.

Reply:
@xlecours ok.